File: erykandroidcommon/src/main/java/pl/edu/amu/wmi/erykandroidcommon/ui/rating/RateCountPair.java
package pl.edu.amu.wmi.erykandroidcommon.ui.rating;


import org.greenrobot.greendao.annotation.Id;

import lombok.Data;
import lombok.Getter;

public class RateCountPair {

    @Id
    private Integer id;

    @Getter
    private int count;

    @Getter
    private float rating;

    private float your;

    private VoteType voteType;

    public enum VoteType {
        GYM, TRAINER
    }

}

File: erykandroidcommon/src/main/java/pl/edu/amu/wmi/erykandroidcommon/ui/rating/RateCountPair.kt
package pl.edu.amu.wmi.erykandroidcommon.ui.rating


import lombok.Data

@Data
class RateCountPair {

    private val id: Int? = null

    private val count: Int = 0

    private val rating: Float = 0.toFloat()

    private val your: Float = 0.toFloat()

    private val voteType: VoteType? = null


}
